hepcrawl/parsers/arxiv.py

```diff
@@ -357,5 +346,5 @@ def source(self):
     @property
     def authors_and_collaborations(self):
         if not hasattr(self, '_authors_and_collaborations'):
-            self._authors_and_collaborations = self._get_authors_and_collaborations(self, self.root)
+            self._authors_and_collaborations = self._get_authors_and_collaborations(self.root)
```

Comment:
Hi @tsgit, just noticed this PR was still open. This seems like a huge mistake that would cause an exception due to wrong number of arguments previously. Do you understand how it was working?

Reply:
Thanks for the IRL explanation!